pkg/tempopb/tempo.proto

```diff
@@ -6,7 +6,9 @@ import "trace/v1/trace.proto";
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 
 service Pusher {
-  rpc PushBytes(PushBytesRequest) returns (PushResponse) {};
+  // different versions of PushBytes expect the trace data to be pushed in different formats
+  rpc PushBytes(PushBytesRequest) returns (PushResponse) {};   // ./pkg/model/v1
+  rpc PushBytesV2(PushBytesRequest) returns (PushResponse) {}; // ./pkg/model/v2
```

Comment:
Are you adding both versions here to allow handling of both versions at the same time?

Reply:
yup. while the distributors transition to the new version the ingesters will need to be able to take both v1 and v2.